src/NumericalAlgorithms/Spectral/SwshInterpolation.hpp

```diff
@@ -33,13 +33,24 @@ class SpinWeightedSphericalHarmonic {
 
   SpinWeightedSphericalHarmonic(int spin, size_t l, int m) noexcept;
 
-  /// Return by pointer the values of the spin-weighted spherical harmonic
-  /// evaluated at `theta` and `phi`.
-  ///
-  /// \details The additional values `sin_theta_over_2` and `cos_theta_over_2`,
-  /// representing \f$\sin(\theta/2)\f$ and \f$\cos(\theta/2)\f$ are taken as
-  /// required input to improve the speed of evaluation when called more than
-  /// once.
+  /*!
+   *  Return by pointer the values of the spin-weighted spherical harmonic
+   *  evaluated at `theta` and `phi`.
+   *
+   *  \details The additional values `sin_theta_over_2` and `cos_theta_over_2`,
+   *  representing \f$\sin(\theta/2)\f$ and \f$\cos(\theta/2)\f$ are taken as
+   *  required input to improve the speed of evaluation when called more than
+   *  once.
+   *
+   *  The formula we evaluate (with various prefactors precomputed, cached, and
+   *  optimized from the factorials) is \cite Goldberg1966uu
+   *  \f{align*}{
```

Comment:
Can you add a blank line before the equation to avoid clang-format being itself? :P